sql/core/src/test/resources/sql-tests/results/ansi/timestamp.sql.out

```diff
@@ -107,6 +107,40 @@ java.time.DateTimeException
 The fraction of sec must be zero. Valid range is [0, 60].
 
 
+-- !query
+SELECT make_timestamp(1, 1, 1, 1, 1, 1)
+-- !query schema
+struct<make_timestamp(1, 1, 1, 1, 1, 1):timestamp>
+-- !query output
+0001-01-01 01:01:01
+
+
+-- !query
+SELECT make_timestamp(1, 1, 1, 1, 1, 60)
+-- !query schema
+struct<make_timestamp(1, 1, 1, 1, 1, 60):timestamp>
+-- !query output
+0001-01-01 01:02:00
+
+
+-- !query
+SELECT make_timestamp(1, 1, 1, 1, 1, 61)
+-- !query schema
+struct<>
+-- !query output
+java.time.DateTimeException
+Invalid value for SecondOfMinute (valid values 0 - 59): 61
+
+
+-- !query
+SELECT make_timestamp(1, 1, 1, 1, 1, null)
+-- !query schema
+struct<>
+-- !query output
+org.apache.spark.sql.AnalysisException
+cannot resolve 'make_timestamp(1, 1, 1, 1, 1, NULL)' due to data type mismatch: argument 6 requires (decimal(8,6) or int) type, however, 'NULL' is of void type.; line 1 pos 7
```

Comment:
What's the previous behavior? also fail?

Reply:
Let's add NullType support.